README.md

```diff
@@ -7,6 +7,39 @@ Clownface is a graph traversal library inspired by [Gremlin](https://tinkerpop.a
 
 Clownface greatly simplifies interacting with RDF data in JavaScript.
 
+# Quick start
+
+The recommended way is to use clownface with and RDF/JS environment. It also requires [`DataFactory`](https://rdf.js.org/data-model-spec/#datafactory-interface) and [DatasetFactory](https://rdf.js.org/dataset-spec/#datasetfactory-interface), for example those provided by [@rdfjs/data-model](https://npm.im/@rdfjs/data-model) and [@rdfjs/dataset](https://npm.im/@rdfjs/dataset) packages respectively, as well as [@rdfjs/namespace](https://npm.im/@rdfjs/namespace).
```

Comment:
Maybe adding a break line after each sentence can help for readability of the Markdown file.

I also added backticks to all links to have something consistent in the whole paragraph.

```suggestion
The recommended way is to use clownface with and RDF/JS environment.
It also requires [`DataFactory`](https://rdf.js.org/data-model-spec/#datafactory-interface) and [`DatasetFactory`](https://rdf.js.org/dataset-spec/#datasetfactory-interface), for example those provided by [`@rdfjs/data-model`](https://npm.im/@rdfjs/data-model) and [`@rdfjs/dataset`](https://npm.im/@rdfjs/dataset) packages respectively, as well as [`@rdfjs/namespace`](https://npm.im/@rdfjs/namespace).
```